cdc/kv/etcd.go

```diff
@@ -111,8 +113,17 @@ type CDCEtcdClient struct {
 }
 
 //NewCDCEtcdClient returns a new CDCEtcdClient
-func NewCDCEtcdClient(cli *clientv3.Client) CDCEtcdClient {
-	return CDCEtcdClient{Client: etcd.Wrap(cli)}
+func NewCDCEtcdClient(ctx context.Context, cli *clientv3.Client) CDCEtcdClient {
+	captureAddr := util.CaptureAddrFromCtx(ctx)
+	metrics := map[string]prometheus.Counter{
+		etcd.EtcdPut:    etcdRequestCounter.WithLabelValues(etcd.EtcdPut, captureAddr),
+		etcd.EtcdGet:    etcdRequestCounter.WithLabelValues(etcd.EtcdGet, captureAddr),
+		etcd.EtcdDel:    etcdRequestCounter.WithLabelValues(etcd.EtcdDel, captureAddr),
+		etcd.EtcdTxn:    etcdRequestCounter.WithLabelValues(etcd.EtcdTxn, captureAddr),
+		etcd.EtcdGrant:  etcdRequestCounter.WithLabelValues(etcd.EtcdGrant, captureAddr),
+		etcd.EtcdRevoke: etcdRequestCounter.WithLabelValues(etcd.EtcdRevoke, captureAddr),
+	}
```

Comment:
IMO, it's better to put the map in package pkg/etcd, and pass capture address to the Wrap.

Reply:
the metrics has cdc specific label `capture`, so I think it is better to put the metrics map in the upper caller.